### DISCUSSION TITLE
Github ipadOs: External monitor

### DISCUSSION THREAD

**Discussion by jeroengui** [community/community#73] [Mobile] [answered]
When I connect an external monitor to my ipad and open the github app, the external monitor displays the home page and uses the full resolution, but It doesn't display anything else. I also can't go there with a mouse attached to my ipad.
 It would be more useful if code could be displayed on the external screen.

**Comment by jeroengui** [comment]
Screenshots for more clarity:
https://ibb.co/nRPwnp0
https://ibb.co/znFnJP1

**Reply by KalleOlaviNiemitalo** [reply]
Got the same problem with AirPlay mirroring from iPhone to Apple TV, in <https://github.com/github/feedback/discussions/3177>.

**Comment by peppy** [comment]
A temporary fix would be great for this (just disabling the flag saying that the application supports external displays so it is correctly mirrored).

**Reply by ali13691369** [reply]
Hi

**Comment by ali13691369** [comment]
Hi

**Comment by eliperkins** [comment, marked as answer]
Thank you for reporting this! We'll address this in an upcoming version of the app. 🙇‍♂️
